integration/main.go

```diff
@@ -38,7 +38,7 @@ import (
 const (
 	integrationTestNamespacePrefix = "istio-ca-integration-"
 	// Specifies how long we wait before a secret becomes existent.
-	secretWaitTime = 20 * time.Second
+	secretWaitTime = 120 * time.Second
```

Comment:
This doesn't look like necessary. 20 seconds are already very generous in a light-loaded cluster and bumping it to 120 seconds will only cover underlying issues/bugs.

@sebastienvas @yutongz @chxchx Did we change the cluster in which tests are run? Particularly did we turn on RBAC in the testing cluster?

Reply:
Yes, we are using rbac cluster in presubmit now. It does reveal a issue and break auth/presubmit. @myidpt and I talked about it this afternoon and I think he is working on fixing.

Reply:
I will revert this value back to 20. I increased to avoid the timeout in e2e.sh script. But it is not related to this value.